charts/consul/templates/server-acl-init-cleanup-job.yaml

```diff
@@ -62,6 +62,14 @@ spec:
             limits:
               memory: "50Mi"
               cpu: "50m"
+      {{- if .Values.global.acls.tolerations }}
+      tolerations:
+        {{ tpl .Values.global.acls.tolerations . | indent 12 | trim }}
```

Comment:
These also don't look like 12 spaces. Why aren't our tests failing? Maybe this is not necessary?

Reply:
We do need it, I will change it to 8. The reason our tests don't fail is that YAML only cares about the level of indent exceeding the current indent.

I tested this by changing the value to 2 and watching it fail because the next indent level was less than the "tolerations:" indent level.

YAML doesn't see the difference between these two:
```yaml
pizzas:
  - one: pepperoni
  - two: cheese
```

```yaml
pizzas:
                   - one: pepperoni
                   - two: cheese
```

That's why the 12 works here. But is is better as 8.

Reply:
Ah that's interesting! Thanks for looking into it!